code/go/0chain.net/smartcontract/stakepool/spenum/enums.go

```diff
@@ -22,6 +22,15 @@ func (p Provider) String() string {
 	return providerString[p]
 }
 
+func ToProviderID(ps string) Provider {
```

Comment:
It's provider type,  so should be `ToProviderType`

Reply:
Sure.